udaykalra/portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,72 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <link rel="stylesheet" href="style.css" />
+
+    <div class="hero-image">
+      <div class="hero-text">
+        <h1>Uday Kalra</h1>
+        <p>
+          Software Engineer <br />
+          :P
+        </p>
+      </div>
+    </div>
+
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+      integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+    <div class="grey">
+      <div class="go animate">
+        <a href="index.html">ME, MYSELF, AND I</a>
+        <a href="Resume.html">RESUME</a>
+        <a href="Projects.html">PROJECTS</a>
+        <a href="Contact.html">CONTACT</a>
+      </div>
+    </div>
+    <div class="container-fluid main-content">
+      <div class="row topRow center">
+        <div class="col-sm-4 center">
+          <img src="face2.jpg" class="rounded-circle" alt="..." style="min-height: 600px; height: 600px;" />
```

Comment:
Can this style be encapsulated within CSS? Especially since the same style is used below, you might get some deduplication with a CSS rule.

Reply:
This has been implemented in latest push.